components/MessageBox/Index.vue

```diff
@@ -45,8 +45,6 @@
 </template>
 
 <script setup>
-let check = true;
-let successMessage1 = "موفقیت آمیز";
-let successMessage2 = "کاربر عزیز اشتراک 12 ماهه طلایی برای شما فعال شد :)";
-let successMessage3 = "از امکانات سگمنتو لذت ببرید.";
+import { defineProps } from "vue";
+const props = defineProps(["message", "check"]);
```

Comment:
It is better to change the `check` property to `isSuccess` and set it as default `true`

Reply:
I'm not agree, this is not true because of the other *props* that are specify the colors and some slots that have get contents and icon. so we have make some components that are using this component as parent like `MessageBoxFail`.

Reply:
Hmmm. Yes I agree, so we should implement the component to something like this

```javascript
<script setup>
const props = defineProps({ 
    color: {
        type: String,
        default: "bg-success"
    },
    title: {
        type: String,
        default: "موفقیت آمیز!" // But later will read data from => assets/config/languages.json
    },
    meta_title: {
        type: String,
        required: true
    }
});
</script>
```

And the template should be like this

```html
<template>
    <div>
        <!-- content -->
        <div>
            <div>{{ title }}</div>
            <div>{{ meta_title }}</div>
            <div><slot name="paragraph"></slot></div>
        </div>
        
        <!-- svg icon -->
        <div>
            <slot name="icon"></slot>
        </div>
    </div>
</template>

```

Reply:
have a look: [review text](https://github.com/xaankoow/segmento-nuxt/pull/42#pullrequestreview-1531773482)

Reply:
Yes. The `title` and `meta_title` defined as props cause we have same style on every `MessageBox` components, so as a developer that I should use this component I shouldn't have permission to change the component concept. But the `paragraph` defined as `slot` cause there is no style and when at some where you doesn't have any style it means you can put anything there. So this could be `slot`.